### ISSUE TITLE
Phone call and video call buttons are not clickable

### ISSUE THREAD

**Issue by thlkmoeller** [telegramdesktop/tdesktop#26597]
### Steps to reproduce

1. Start Telegram
2. Click on the phone symbol on the top right corner
3. Move the mouse over the call or the video call button

### Expected behaviour

the cursor should change its shape and it should be possible to make a call

### Actual behaviour

the cursor does not change its shape, a click on the call or video call button has no effect. Furthermore it is not possible to close the window.

And by accident i found out, that it is possible to click the buttons, when moving the cursor into a position, that is actually below the button. It is really weird and happens in 95 percent of the time. please see screenshots.

And the bug exists already for quite a time. all the updates of Telegram did not fix it. Although i guess, that this is not necessarily a bug of Telegram, because i use two more laptops, with the same Linux Version, but with Mate instead of Cinnamon. There the bug does not exist.

![Bildschirmfoto vom 2023-07-31 09-42-54](https://github.com/telegramdesktop/tdesktop/assets/77677066/94dbf229-dfa2-4a25-bf35-6396fc9e12b2)
![Bildschirmfoto vom 2023-07-31 09-44-32](https://github.com/telegramdesktop/tdesktop/assets/77677066/0ebe7688-3e5a-4ed0-87f1-d4c81fb04434)
![Bildschirmfoto vom 2023-07-31 09-44-13](https://github.com/telegramdesktop/tdesktop/assets/77677066/701e14db-44f7-4b79-b05b-a0ec16f075eb)
![Bildschirmfoto vom 2023-07-31 09-43-54](https://github.com/telegramdesktop/tdesktop/assets/77677066/a7594f09-f6ea-43d2-b2ad-59a484d1e9c3)
![Bildschirmfoto vom 2023-07-31 09-43-26](https://github.com/telegramdesktop/tdesktop/assets/77677066/5908db21-26ec-463b-9ce1-844653f5100e)
![Bildschirmfoto vom 2023-07-31 09-40-00](https://github.com/telegramdesktop/tdesktop/assets/77677066/360f8fd6-2a38-4ba1-9ad6-16c12cb39bb3)

### Operating system

Linux Mint 20.3, Desktop environment: Cinnamon

### Version of Telegram Desktop

4.8.10

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
Yeah, this sounds pretty much like a Cinnamon bug, this is not the first report through, search for the closed ones.

**Comment by thlkmoeller** [comment]
yeah thanks, i found them. i will have a look for the cinnamon bug then somewhere.

The reports here are:
https://github.com/telegramdesktop/tdesktop/issues/25114
https://github.com/telegramdesktop/tdesktop/issues/26023
https://github.com/telegramdesktop/tdesktop/issues/26537

thanks again
